src/main/java/org/mitre/synthea/export/FhirR4.java

```diff
@@ -24,13 +26,10 @@
 import java.util.stream.Collectors;
 
 import org.apache.commons.lang3.StringUtils;
-import org.hl7.fhir.r4.model.Address;
-import org.hl7.fhir.r4.model.AllergyIntolerance;
+import org.hl7.fhir.r4.model.*;
```

Comment:
Thank you for the pull request!

We have not started a formal review yet, but I noticed the change to star-imports.

Please do not use star-imports -- they violate our style guidelines, and should appear as WARN messages when you run the unit tests (which incidentally are failing).

Reply:
hey @jawalonoski 
Thanks for looking into it.

I just created the PR for others to share the code.
I wasnt intending to merge this  😅 
Thats the reason I left it in draft.

I will try cleaning this up if you feel its worth pursuing.

Reply:
this whole PR is a mess. 🤦 
I ended up merging some Consent related changes in there as well which was me experimenting Consent generation per Patient which snuck into this PR.

I can close it out if it adds too much noise for you.